### PR TITLE
Azure Functions SqlJobFlow implementation

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
Hi @rguthriemsft, 

I implement the Azure Functions. But not tested yet. I'm waiting for the proctor environment deployment script.  If you don't like the design, please let me know. I'm going to flight from now. 
After arriving Japan, I start testing it.

### CODE REVIEW

File: leaderboard/batch/Batch/SqlJobFlow.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    public static class SqlJobFlow
+    {
+        private static SqlConnection connection;
+        private static string queryString = "some batch execution sql";
+
+        static SqlJobFlow()
+        {
+            var connectionString = Environment.GetEnvironmentVariable("SqlConnectionString");
+            connection = new SqlConnection(connectionString);
+            connection.Open();
```
Comment (rguthriemsft) [inline leaderboard/batch/Batch/SqlJobFlow.cs]: You might need a try/catch here so you can log error to console if the conn and execute is failing for whatever reason.

File: leaderboard/batch/helm/Readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Leaderboard barch chart
```
Comment (dtzar) [inline leaderboard/batch/helm/Readme.md]: barch --> batch

File: leaderboard/batch/Batch/SqlJobFlow.cs (hunk 1 above)
Comment (TsuyoshiUshio) [reply leaderboard/batch/Batch/SqlJobFlow.cs]: Thanks. Done.

File: leaderboard/batch/helm/Readme.md (hunk 2 above)
Comment (TsuyoshiUshio) [reply leaderboard/batch/helm/Readme.md]: Thanks done.

**Comment by TsuyoshiUshio** [comment]
Hi @rguthriemsft and @dtzar , 

I tested this azure functions run on the kubernetes with helm and works.  I test the build_deploy_leaderboard_batch.sh by manually.

https://github.com/Azure-Samples/openhack-devops-proctor/blob/feature/batch/provision-proctor/build_deploy_leaderboard_batch.sh

Since I'm waiting for the proctor environment deployment script, I can't fully tested (especially kvstore part) however, I tested other parts manually. This script never affect other scripts, so go ahead merge this. If you guys are OK.

**Comment by TsuyoshiUshio** [comment]
Hi @dtzar , 

I add the script to setup.sh. Go ahead and MERGE! :)

**Pull request merged**